apps/src/aichat/views/ChatWorkspace.tsx

```diff
@@ -226,4 +225,24 @@ const ChatWithModel: React.FunctionComponent<ChatWithModelProps> = ({
   );
 };
 
+interface StudentChatHistoryViewProps {
+  events: ChatEvent[];
+}
+
+const StudentChatHistoryView: React.FunctionComponent<
+  StudentChatHistoryViewProps
+> = ({events}) => {
+  return (
+    <div
+      id="student-chat-history-workspace"
+      className={moduleStyles.conversationArea}
+    >
+      {events.map((event, index) => {
+        event = {...event, isTeacherView: true};
+        return <ChatEventView event={event} key={index} />;
+      })}
```

Comment:
keys on `index` values are frowned upon. Is there something in the `event` object we could key off of instead? Maybe `event.timestamp`? I'm unclear if that's unique enough. A combo of `timestamp + descriptionKey`?

Reply:
Thanks! I think `timestamp` would be unique enough so will update.